lib/index.js

```diff
@@ -96,6 +96,36 @@ class LibConsoleCLI {
     return projects
   }
 
+  /**
+   * Retrieves project details from the Console APi
+   *
+   * @param {*} orgId Organization AMS ID
+   * @param {*} projectId Console project ID
+   * @returns {object} Project metadata
+   */
+  async getProject (orgId, projectId) {
+    spinner.start('Getting Project...')
+    const project = (await this.sdkClient.getProject(orgId, projectId)).body
+    spinner.stop()
+    logger.debug(`Get Project response for Org ${orgId}: ${JSON.stringify(project, null, 2)}`)
+    return project
+  }
+
+  /**
+   * Retrieves application extensions from the extension registry API
+   *
+   * @param {*} orgId Organization AMS ID
+   * @param {*} appId App Builder application ID
+   * @returns {object} Extension point metadata
+   */
+  async getApplicationExtensions (orgId, appId) {
```

Comment:
How does this differ from https://github.com/adobe/aio-cli-lib-console/blob/master/lib/index.js#L591= ?

Reply:
This function makes use of [a new function in aio-lib-console added in another PR](https://github.com/adobe/aio-lib-console/blob/41f17b96a0bd87e6c72cfe4be671c267fb4a700c/src/index.js#L1203) that hits an endpoint that provides the extension's `status`. The function linked above hits an endpoint that does not.

Reply:
got it thanks